src/bijectors/composed.jl

```diff
@@ -225,10 +225,10 @@ end
     for i = 2:length(T.parameters)
         temp = gensym(:temp)
         push!(expr.args, :($temp = forward(cb.ts[$i], y)))
```

Comment:
Again, another `forward` call above and here:
```suggestion
        temp = gensym(:res_logjac)
        push!(expr.args, :(y, $temp = with_logabsdet_jacobian(cb.ts[$i], y)))
```